ui/lib/pki/addon/components/pki-key-form.js

```diff
@@ -46,6 +46,11 @@ export default class PkiKeyForm extends Component {
     }
   }
 
+  @action
+  onFileUpload() {
+    // do something
```

Comment:
Just wanted to make sure this action isn't supposed have anything yet?

Reply:
🤔  nope, I definitely added something - maybe I'm missing a commit

Reply:
Hmm - it's showing up on my end! Maybe you were so speedy it was an outdated github version?!

Reply:
OOH 🤣 this should be deleted, all of the import functionality is handled by the `pki-key-import.js` component. Holiday brain 🌴 

nice catch!